jet/dashboard/templates/jet.dashboard/modules/recent_actions.html

```diff
@@ -25,7 +25,7 @@
                     {% endif %}
 
                     {% if entry.content_type %}
-                        <span>{% filter capfirst %}{{ entry.content_type }}{% endfilter %}</span>
+                        <span>{% filter capfirst %}{{ entry.content_type.name }}{% endfilter %}</span>
```

Comment:
So that's how you fixed that recent actions label! Nice find

Reply:
yeah the default str returned for content_type changed, so switched to .name to get the same result